modules/geo-layers/src/mvt-layer/mvt-layer.js

```diff
@@ -216,12 +216,29 @@ export default class MVTLayer extends TileLayer {
     return renderedFeatures;
   }
 
+  getVisibleTiles() {
+    return this.state.tileset.selectedTiles;
+  }
+
+  _convertTilesToWGS84(tiles, viewport) {
```

Comment:
This method is not performing any conversion. Call it something like "setupTileTransform"